js/views/roomlistview.js

```diff
@@ -154,9 +154,21 @@
 			});
 		},
 		templateContext: function() {
+			var icon = '';
+			if (this.model.get('objectType') === 'file') {
```

Comment:
switch?
Much cleaner ;)

Reply:
doesnt work, because its type and objectType that is being checked?